.github/workflows/ci.yml

```diff
@@ -134,3 +134,34 @@ jobs:
 
     - name: Check Formatting
       run: git diff --exit-code
+
+  tidy:
+    name: Analyzing on ${{ matrix.platform.name }}
+    runs-on: ${{ matrix.platform.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+        - { name: Linux, os: ubuntu-latest }
+        - { name: macOS, os: macos-12 }
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+
+    - name: Install Linux Dependencies
+      if: runner.os == 'Linux'
+      run: sudo apt-get update && sudo apt-get install libxrandr-dev libxcursor-dev libudev-dev libopenal-dev libflac-dev libvorbis-dev libgl1-mesa-dev libegl1-mesa-dev libdrm-dev libgbm-dev
```

Comment:
Installing `libdrm-dev` and `libgbm-dev` is not needed as long as option `SFML_USE_DRM` is false.

Reply:
That makes me wonder if I should go ahead and add clang-tidy + DRM jobs since currently those source files are not getting analyzed.